### PR TITLE
Update slevomat/coding-standard and rector

### PR DESCRIPTION
**Pull request by alexndlm**
```diff
! Required php8.1
! Required Symfony 6.1
```

- Add to ECS EonX set:
  - OrderedClassElementsFixer
  - DisallowTrailingCommaInCallSniff
  - DisallowTrailingCommaInClosureUseSniff
  - DisallowTrailingCommaInDeclarationSniff
- Remove SortConstantsAlphabeticallyRector, SortMethodsAlphabeticallyRector, SortPropertiesAlphabeticallyRector due to https://github.com/rectorphp/rector-src/pull/1910
- We can use the new rector configs format https://getrector.org/blog/new-in-rector-012-introducing-rector-config-with-autocomplete
- We can use the new ECS  configs format
- PhpStan - https://phpstan.org/blog/solving-phpstan-error-unsafe-usage-of-new-static

### CODE REVIEW

**Review by roman-eonx — CHANGES_REQUESTED** [review]
- Let's also update PHPStan to the latest release, as there are many improvements in 1.7.x.
- Let's update Rector to the latest release (0.13.x).

**Comment by alexndlm** [comment]
> * Let's also update PHPStan to the latest release, as there are many improvements in 1.7.x.
> * Let's update Rector to the latest release (0.13.x).

Have done.

File: src/Rector/PhpDocCommentRector.php [hunk 1]
```diff
@@ -260,17 +265,15 @@ private function checkTextNode(PhpDocTextNode $phpDocTextNode): void
             $text[$lastKey] .= '.';
         }
 
-        $linesCount = \count($text);
-        for ($iterator = 1; $iterator < $linesCount; $iterator++) {
-            $text[$iterator] = ' * ' . $text[$iterator];
-        }
-
+        // We need generate new text without * for compare.
```
Comment (roman-eonx) [inline src/Rector/PhpDocCommentRector.php:268]: Please update the comment:
```
// We need to generate new text without "*" for comparison
```

**Review by roman-eonx — CHANGES_REQUESTED** [review]

File: src/Rector/PhpDocCommentRector.php (hunk 1 above)
Comment (alexndlm) [reply src/Rector/PhpDocCommentRector.php]: Have done

File: src/Rector/SingleLineCommentRector.php [hunk 2]
```diff
@@ -113,7 +115,7 @@ private function checkComments(array $comments): array
             $disallowEnding = $this->checkLineEndingDisallowed($commentText);
 
             if ($disallowEnding !== null) {
-                $pattern = '#' . \preg_quote($disallowEnding) . '$#';
+                $pattern = '#' . \preg_quote($disallowEnding, '#') . '$#';
```
[the remark this answers is not in the document]
Comment (alexndlm) [reply src/Rector/SingleLineCommentRector.php:118]: If you talking about my change:
Снимок экрана 2022-06-14 в 11 18 07

**Review by akl-95 — APPROVED** [review]

**Review by ERuban — APPROVED** [review]

**Pull request merged**